client/allocrunner/networking_bridge_linux.go

```diff
@@ -134,8 +138,34 @@ func (b *bridgeNetworkConfigurator) Teardown(ctx context.Context, alloc *structs
 	return b.cni.Teardown(ctx, alloc, spec)
 }
 
-func buildNomadBridgeNetConfig(bridgeName, subnet string) []byte {
-	return []byte(fmt.Sprintf(nomadCNIConfigTemplate, bridgeName, subnet, cniAdminChainName))
+func buildNomadBridgeNetConfig(b bridgeNetworkConfigurator) []byte {
+	tmpl, err := template.New("cniConf").Parse(nomadCNIConfigTemplate)
+
+	// TODO: Consider exporting these directly from bridgeNetworkConfigurator
+	// so they aren't repeated in the input struct
+	type templInput struct {
+		AllocSubnet       string
+		BridgeName        string
+		HairpinMode       bool
+		CNIAdminChainName string
+	}
+
+	tIn := templInput{
+		AllocSubnet:       b.allocSubnet,
+		BridgeName:        b.bridgeName,
+		HairpinMode:       b.hairpinMode,
+		CNIAdminChainName: cniAdminChainName,
+	}
+
+	if err != nil {
+		panic(err)
+	}
```

Comment:
I'd move this err check to right under its declaration above the templInput stuff, and maybe add a comment about why panic() is ok/desirable vs returning an error up the stack?